test/Sentry.Maui.Device.TestApp/Platforms/Android/SentryInstrumentation.cs

```diff
@@ -0,0 +1,46 @@
+using Android.App;
+using Android.Runtime;
+using DeviceRunners.XHarness.Maui;
+using Environment = System.Environment;
+
+namespace Sentry.Maui.Device.TestApp;
+
+/// <summary>
+/// Decorating this class with the Instrumentation attribute means we can specify it as our instrumentation entry point
+/// when launching from XHarness. We can then intercept the OnStart event to set environment variables on Android from
+/// arguments passed to the instrumentation. Note that AndroidEnableMarshalMethods also has to be set in the csproj file
+/// for this to work.
+/// </summary>
+[Instrumentation(Name = "Sentry.Maui.Device.TestApp.SentryInstrumentation")]
+public class SentryInstrumentation : XHarnessInstrumentation
+{
+    protected SentryInstrumentation(IntPtr handle, JniHandleOwnership ownership)
+        : base(handle, ownership)
+    {
+    }
+
+    public override void OnStart()
+    {
+        try
+        {
+            Console.WriteLine("Parsing instrumentation arguments");
+            if (IsGitHubActions)
+            {
+                Console.WriteLine("CI build detected - setting environment variables for CI on the device");
+                Environment.SetEnvironmentVariable("GITHUB_ACTIONS", "true");
+            }
+            else
+            {
+                Console.WriteLine("CI build not detected - no environment variables set");
+            }
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine(e);
```

Comment:
Failure here would only be calling `IsGitHubActions` or `SetEnvironmentVariable` ?
Is this common? since we supresss the exception and continue, I wonder as it'll happen without us noticing unless we read the right part of the build output

Reply:
We could probably remove this now 👍🏻 

It was always throwing an exception until I added this:
https://github.com/getsentry/sentry-dotnet/blob/4780038a8bf422d0190c0c1e75f9c9dcf38c02e0/test/Sentry.Maui.Device.TestApp/Sentry.Maui.Device.TestApp.csproj#L19-L20